Add marketing processes to the Handbook.md (#133)
flowforge/handbook: design/index.md

Blender is a free and open-source 3D computer graphics software toolset. It can be used to create static renders of 3d assets, or entire 2d & 3d animations. 

## Websites

- All written content should be in UK English.
- All page titles should summarise the content, keep the URL length as short as practical and use [Kebab Case](https://en.wiktionary.org/wiki/kebab_case).
- All images should use informative [alt tags](https://www.w3.org/WAI/tutorials/images/tips/) which clearly describe the point of an image rather than all the details. Alt tags should be no longer than 60 characters.
- When mentioning [FlowForge Concepts](https://flowforge.com/docs/user/concepts/) (terminology) where possible we should link to an explanation of that concept.

## Social Media, Marketing & PR

> IT is important we understand what we communicate, to who, how and when. This sets out our principles for social media, marketing and PR communications.

Reviewer: ```suggestion
It's important we understand what we communicate, to who, how, and when. This sets out our principles for social media, marketing, and PR communications.
```

### Content Types

> Content should be published to the appropriate channels, the content we create is as follows.

Reviewer 2: ```suggestion
Content should be published to the appropriate channels, the types of published content we currently produce are as follows.
```

#### Release

Major releases of FlowForge, the timing of these releases are governed by our [development cadence](https://flowforge.com/handbook/development/#cadence).

#### Patch

Patch between releases, these will happen as needed and may be at short notice.

#### Newsletter

Monthly newsletter published at the start of each calendar month covering news from the prior month.

#### How To

Technical guides, published once per week.

> #### Planned Downtime

Author: I've updated the order as per this suggestion.

As soon as we are aware of planned downtime we should publish the information, that should be at least one week before the downtime.

#### Outage

If we experience unplanned downtime we should communicate in near real time to keep our users and stakeholders up to date with the situation.

#### Corporate Comms

> Major company news such as investment rounds should be communicated at the time agreed with the relevant parties involved, it is important there is no significant delay in communicating this information.

Reviewer: ```suggestion
Major company news such as investment rounds should be communicated at the time agreed with the relevant parties involved. We strive to make this information public as soon as practical and effective.
```

This is ambiguous as a sentence, does our audience not experience a delay, or inter party communication?

### Content Channels

Each content type has channels which are appropriate, this table sets out which content should be posted on each channel.

|Content          |Blog  |Mailing List |Twitter  |Youtube  |LinkedIn |Reddit |Node-RED Slack|
|---              |---   |---          |---      |---      |---      |---    |---           |
|Release          |*     |             |*        |*        |*        |*      |*             |  

> |Patch            |      |             |*        |         |         |*      |*             |   

Reviewer: I think a patch release should require a blog post. Unlike the release blog post, the tone is more factual and much shorter. But it does need publishing what was fixed and improved. These changes might require specific update instructions too.

Author: I've made this change.

|Newsletter       |*     |*            |*        |         |*        |*      |*             |
|How To           |      |*            |*        |         |*        | *     |*             |  

> |Planned Downtime |      |*            |*        |         |*        |*      |*             |  

Reviewer: ```suggestion
|Planned Downtime | *     |?            |*        |         |*        |*      |*             |  
```

I think our mailing list is the highlights of the past 30 days or so. Planned downtime might occur in before a newsletter goes out too?

Or is the Mailing list not the same as the monthly newsletter to the mailing list?

Author: Sorry, this was an error in my MD table, I don't think we should send emails about planned downtime.

Author: Hopefully resolved in https://github.com/flowforge/handbook/pull/133/commits/a01d7df2db115203663ec7ed363ee0049aecdb30

|Outage           |      |             |         |         |         |       |              |
|Corporate Comms  |*     |             |*        |         |*        |       |              |

### Channel Content Guidance

#### Blog

Content should be eloquent and an appropriate length to communicate the message.

#### Mailing List

Content should be eloquent and an appropriate length to communicate the message.

#### Twitter

> Keep it short, Tweets should be factual, informal and have a call to action (a link) where a user can read more.

Reviewer: I'd also say this is an opportunity to be more fun and not too corporate in tone. A blog is more formal for example.

#### Youtube

Videos should:

- Include opening and closing idents
- Be clear in what they are communicating.
- Have an appropriate thumbnail.
- Have a call to action in the description.
- Be publicly listed.

> 

Reviewer 2: Further details will be found in the (to be reviewed/merged) `/handbook/media` section

#### LinkedIn

> Posts should be framed in the context of FlowForge as an organisation and a team, don't get too technical. A relevant photo or image should be included in every post. Where the post is primarily of a video it should be embedded from Youtube.

Reviewer: 👌

#### Reddit

> Keep it short, Reddit posts should be factual, informal and have a call to action (a link) where a user can read more.

Author: I've made this change.
